Resources/Prototypes/Catalog/Fills/Paper/manuals.yml

```diff
@@ -16,3 +16,40 @@
       Higher will burn the engine out and eventually make it explode. Don't.
       Don't forget to refuel it, it tends to stop at the worst possible time.
 
+- type: entity
+  id: SecCode
+  parent: PaperWritten
+  name: S.E.C pocketbook
+  description: An abridged version of the Secure Enviroment Conduct manual or S.E.C for short. A must-read for all security personnel!
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/bureaucracy.rsi
+    netsync: false
+    layers:
+    - state: seccode
+  - type: Paper
+    content: |
+      DISCLAIMER: S.E.C codes are not to be treated as gospel, as they are a amalgamation of general suggestions of how to run security on a commercial spacecraft.
+      However, violating them should be generally avoided, and abuse will be punished accordingly.
+      
+      PROPER ARRESTS:
+      In most situations a non-lethal option is both more effective and humane. You may not harm crewmembers not causing direct harm to you.
+      In order to detain a crewmember:
+      1. Subdue the offender (via your stun baton or taser) and handcuff them.
+      2. Bring them into the interregation room, you may stripsearch them for contraband there
+      3. Prove the accused of a crime, via collecting evidence from reliable sources
+      4. You may write a document, detailing the code of the crime, the criminal and the accusing party.
+      5. Punish according to the code. Track the time on your stopwatch.
```

Comment:
Doesn't detaining them seem more appropriate given we don't have interrogation rooms.
4. doesn't really make sense.
5. Seems more appropriate to not mention the stopwatch and just tell them to let them free at an appropriate time because realistically it's not going to be tracked if even more than 1 person is arrested.

Reply:
Closing this until further resolution over on discord.
Leave a comment with the results on the PR or ping me